config/samples/infra_v1alpha1_clusterscaleprofile.yaml

```diff
@@ -11,6 +13,8 @@ apiVersion: infra.nephio.org/v1alpha1
 kind: ClusterScaleProfile
 metadata:
   name: medium-fixed-site
+  annotations:
```

Comment:
I don't think we need this annotation on the in-cluster instance of the resource, only on the in-package placeholder.

Reply:
Removed